modules/layers/src/scatterplot-layer/scatterplot-layer-vertex.glsl.js

```diff
@@ -24,45 +24,53 @@ export default `\
 attribute vec3 positions;
 
 attribute vec3 instancePositions;
-attribute vec2 instancePositions64xyLow;
 attribute float instanceRadius;
 attribute vec4 instanceColors;
+attribute vec4 instanceStrokeColors;
 attribute vec3 instancePickingColors;
 
 uniform float opacity;
 uniform float radiusScale;
 uniform float radiusMinPixels;
 uniform float radiusMaxPixels;
+uniform float stroke;
 uniform float outline;
 uniform float strokeWidth;
 
 varying vec4 vColor;
+varying vec4 bColor;
```

Comment:
We use the `v` prefix to indicate that it is a varying. Maybe call this `vStrokeColor` or similar?